src/coreComponents/mesh/MeshLevel.cpp

```diff
@@ -348,7 +348,10 @@ void MeshLevel::generateAdjacencyLists( arrayView1d< localIndex const > const &
       }
       for( localIndex const & edi: elem2dToEdges[ei] )
       {
-        edgeAdjacencySet.insert( edi );
+        if( edi > -1 )
```

Comment:
what's the meaning of -1?

Reply:
That was indeed not needed. I've removed that.